libc/src/math/generic/CMakeLists.txt

```diff
@@ -3601,3 +3601,15 @@ add_entrypoint_object(
   COMPILE_OPTIONS
     -O3
 )
+
+add_entrypoint_object(
+  f16sqrtf
+  SRCS
+    f16sqrtf.cpp
+  HDRS
+    ../f16sqrtf.h
+  DEPENDS
+    libc.src.__support.FPUtil.sqrt
```

Comment:
also `libc.src.__support.macros.properties.types`

Reply:
I'll open a PR later to fix it for the other `*f16` functions above too.